src/Symfony/Component/Finder/Finder.php

```diff
@@ -614,6 +614,20 @@ public function append($iterator)
     }
 
     /**
+     * Check if the any results were found.
+     *
+     * @return bool
+     */
+    public function found()
+    {
+        foreach ($this->getIterator() as $null) {
```

Comment:
I see that as outside the scope of this change, I suggest opening a separate PR if you believe that should be changed